r/bigquery/README.md

```diff
@@ -31,7 +31,7 @@ limitations under the License.
 1. Install R packages `adbcdrivermanager` and `arrow`:
 
    ```r
-   install.packages(c("adbcdrivermanager", "arrow"))
+   install.packages(c("adbcdrivermanager", "tibble", "arrow"))
```

Comment:
should we update the text above this too ("Install R packages `adbcdrivermanager` and `arrow`:") so the lists of packages is the same??

Reply:
Thanks, updated.